seeders/instructors.seeder.js

```diff
@@ -3,11 +3,62 @@ import Instructor from "../src/models/Instructor";
 import User from "../src/models/User";
 
 const data = async () => {
-  const albertBaker = await User.findOne({ email: "instructor@example.com", accessLevel: "instructor" });
+  const chiaChuChiang = await User.findOne({
```

Comment:
Yeah, very much agree here. We should keep all test emails using the `@example.com` since that's domain is reserved for testing. Might also want to add something to the production path to avoid sending with that domain, though not sure what Sendgrid's default behavior is there.

I like the direction of using something like [Faker](https://github.com/marak/Faker.js/) to avoid any references to specific "real-world" accounts in the codebase.

Reply:
Email accounts have been fixed to @example.com